Documentation/api.md

```diff
@@ -226,7 +226,7 @@ NO+p24BL3PHZyKw0nsrm275C913OxEVgnNZX7TQltaweW23Cd1YBNjcfb3zv+Zo=
 
 ## Assets
 
-If you need to serve static assets (e.g. kernel, initrd), `matchbox` can serve arbitrary assets from the `-assets-path`.
+If you need to serve static assets, such as kernel, initrd, and so on, `matchbox` can serve arbitrary assets from the `-assets-path`.
```

Comment:
Recommend: "Matchbox can serve arbitrary static assets such as kernel and initrd pairs from the `-assets-path`."

And with that, let's stop fiddling with this sort of subtle rewrite and go back to the scope of this PR: Headings title case and use of Matchbox as a proper noun.